File: Assignments/Assignment_64/Program_1.java
/*
Enter the number:
16384
15th bit is ON

Enter the number:
16348
15th bit is OFF*/
import java.util.*;

class program_1
{
    public static void main(String[] A) 
    {
        int iNo = 0, iMask = 0, iResult = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number: ");
        iNo = sobj.nextInt();

        iMask = 0x00004000;

        iResult = iNo & iMask;

        if(iResult == iMask)
        {
            System.out.println("15th bit is ON");
        }
        else
        {
            System.out.println("15th bit is OFF");
        }
    }
}

File: Assignments/Assignment_76/Program_1.c
import java.util.Scanner;

class OS
{
    public void Warning(int Battery)
    {
        if(Battery <= 5)
        {
            System.out.println("Status : Critical");
        }
        else if(Battery <= 15)
        {
            System.out.println("Status : Low");
        }
        else
        {
            System.out.println("Status : Normal");
        }
    }
}

class program_1
{
    public static void main(String A[]) 
    {
        Scanner sobj = new Scanner(System.in);
        
        int Battery = 0;

        System.out.print("Enter the Battery Helth : ");
        Battery = sobj.nextInt();

        if((Battery < 0) && (Battery >= 100))
        {
            System.out.println("Invalid Input !");
            return;
        }
        
        OS oobj = new OS();
        oobj.Warning(Battery);

        System.out.println("Battery Percentage : "+Battery);

        sobj.close();
    }
}
